src/resources/dart-3-migration.md

```diff
@@ -596,6 +596,14 @@ analyzer:
   have been cleaned up [#50700](https://github.com/dart-lang/sdk/issues/50700).
 * The output of `dart format` changed a bit for
   [some code](https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#formatter).
+* Windows users with a default pub-cache in `%APPDATA%\Pub\Cache` will get a
+  new empty pub-cache in `%LOCALAPPDATA%\Pub\Cache`.
+  Starting Dart 2.8 a pub-caches would be created `%LOCALAPPDATA%\Pub\Cache`,
+  unless an existing pub-cache was found in `%APPDATA%\Pub\Cache`.
+  As of Dart 3 existing pub-caches in `%APPDATA%\Pub\Cache` will no longer be
+  used. Thus, users with such as pub-cached prior to Dart 2.8 will get a new
+  empty pub-cache. If you have `%APPDATA%\Pub\Cache\bin` in your `PATH` may wish
+  to update your `PATH` variable.
```

Comment:
**issue:** 
Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
* The default location for pub-cache in Windows changed in Dart 2.8.
  It changed from `%APPDATA%\Pub\Cache` to `%LOCALAPPDATA%\Pub\Cache`.
  Between Dart 2.8 and 2.19, Dart used the pub-cache found in the old location.
  If a pub-cache wasn't found in the old location,
  Dart created an empty one in the new location.
  Dart 3 expects the pub-cache to exist in the new location.
  If one doesn't exist in the new location,
  Dart 3 creates a new, empty pub-cache in the new location.
  Dart 3 ignores any pub-cache in the old location.
  Check your `PATH` for `%APPDATA%\Pub\Cache\bin`.
  If that value exists, change it to `%LOCALAPPDATA%\Pub\Cache`.
```

Reply:
I did a little rewrite, credits @whesse, who suggested we omitted a bunch of details.

Indeed there is no reason to explain when stuff happened.